src/pyseekdb/utils/embedding_functions/litellm_embedding_function.py

```
@@ -168,18 +163,12 @@ def __call__(self, input: Documents) -> Embeddings:
         else:
-            raise ValueError(
-                f"Unexpected response format from LiteLLM: {type(response)}"
-            )
+            raise ValueError(f"Unexpected response format from LiteLLM: {type(response)}")
 
         # Validate that we got the expected number of embeddings
         if len(embeddings) != len(input):
-            raise ValueError(
-                f"Expected {len(input)} embeddings but got {len(embeddings)} from LiteLLM"
-            )
+            raise ValueError(f"Expected {len(input)} embeddings but got {len(embeddings)} from LiteLLM")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)